charts/seqr/templates/cronjob.yaml

```diff
@@ -29,7 +29,7 @@ spec:
             deployment: {{ $.Release.Name }}
             {{- include "seqr.labels" $ | nindent 12 }}
         spec:
-          activeDeadlineSeconds: 1800 # 30 Minutes?  TODO: get a real number here.
+          activeDeadlineSeconds: {{ $c.timeout_s | default 1800 }}
```

Comment:
This works... but we should also consider making the job faster (I think there's lots of non-concurrent network requests).  But that's much bigger lift.

Reply:
Yeah, I agree, fixing the job is out of scope here -- this change on its own is valuable in that allows people to specify individual cron job timeouts for each job they create 👍

Reply:
agreed that this job could be improved, but that its out of scope for this